packages/ra-ui-materialui/src/layout/Confirm.tsx

```diff
@@ -94,12 +96,16 @@ const Confirm: FC<ConfirmProps> = props => {
                 {translate(title, { _: title, ...translateOptions })}
             </DialogTitle>
             <DialogContent>
-                <DialogContentText>
-                    {translate(content, {
-                        _: content,
-                        ...translateOptions,
-                    })}
-                </DialogContentText>
+                {isContentString ? (
```

Comment:
I'm not handling this conditional logic in a very elegant way. I wasn't sure how best you'd like this done. The two solutions I have in mind are 

1. Leave it as it is 

2. Do somethng like this
```
import MUIContent from '@material-ui/blahblah`

function Content(props) {
  if (typeof props.content === 'string') {
    return (
      <DialogContentText>
        {translate(content, {
          _: content,
         ...translateOptions,
        })}
    </DialogContentText>
  }

  return props.content
}

function Confirm() {
  
  return (
    <Dialog {..existingProps}> 
      {otherElements}
      <Content />
      {otherElements}
    </Dialog>
  )
}
```

Or any other suggestions you folks might have